### ISSUE TITLE
Question about the assert in loader.py

### ISSUE THREAD

**Issue by Kowasaki** [thtrieu/darkflow#338]
I was wondering what this assert is checking for: https://github.com/thtrieu/darkflow/blob/master/darkflow/utils/loader.py#L75 

I have a custom trained model and config file from darknet and when I tried use prediction I keep getting the "expected x bytes, found y" error. If I comment it out then darkflow runs fine, but I also am unsure what issues that would cause?

**Comment by Kowasaki** [comment]
Hmmm well it looks like now the detection can't find anything. Still wondering what walk.offset is though.

**Comment by minhnhat93** [comment]
Hi, I got the same problem too. Just go to this file https://github.com/thtrieu/darkflow/blob/479c83e14559fd5eceb9a9f612503b29a67fac5c/darkflow/utils/loader.py#L121 and change to `self.offset = 20` it will work

**Comment by Kowasaki** [comment]
@minhnhat93 You're right it worked! Thanks so much! Interestingly, the results I got from darkflow detection is slightly different from darknet using the same weights. Not sure if it has anything to do with the offsets though.

**Comment by michaeldong** [comment]
after change  self.offset = 16  ==> self.offset =20, You need uninstall darkflow, and pip3 install it.
